src/python/grpcio/grpc/experimental/aio/_metadata.py

```diff
@@ -37,6 +37,12 @@ def __init__(self, *args: Tuple[MetadataKey, MetadataValue]) -> None:
         for md_key, md_value in args:
             self.add(md_key, md_value)
 
+    @classmethod
+    def from_tuple(cls, raw_metadata: tuple):
+        if raw_metadata:
```

Comment:
What's the reasoning behind the conditional here? Shouldn't `return cls(*raw_metadata)` just work?

Reply:
Yes, but ``raw_metadata`` can be None.
I'll change it to be explicit, like

```python
if raw_metadata is None: ...
```